app/assets/javascripts/discourse/app/components/bread-crumbs.js

```diff
@@ -10,21 +10,8 @@ export default Component.extend({
   editingCategory: false,
   editingCategoryTab: null,
 
-  @discourseComputed("categories")
-  filteredCategories(categories) {
-    return categories.filter(
-      (category) =>
-        this.siteSettings.allow_uncategorized_topics ||
-        category.id !== this.site.uncategorized_category_id
-    );
-  },
-
-  @discourseComputed(
-    "category.ancestors",
-    "filteredCategories",
-    "noSubcategories"
-  )
-  categoryBreadcrumbs(categoryAncestors, filteredCategories, noSubcategories) {
```

Comment:
I am not sure I follow the change here, why do we need to remove `filteredCategories` here?

Reply:
We used to filter categories here and in d-navigation, the component that renders this one. I moved all the filtering logic there, to keep dependencies between components simpler.